scripts/update_authors.pl

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env perl
```

Comment:
Let's add this script to `dist_noinst_SCRIPTS` section of `scripts/Makefile.am`.